.gitignore

```diff
@@ -8,5 +8,7 @@
 data/data/
 
 # Executables
-./magic
-./lbxdump
+lbxdump
+lbxviewer
+magic
```

Comment:
For me I need to put `/` at the front to ensure they only ignore the binaries. Otherwise just `magic` by itself will ignore everything under `game/magic`. Do you see a different behavior?

Reply:
Yes, with MacOS and git `2.43.0`  `./magic` does not work, but `/magic` would.